.github/workflows/precommit.yml

```diff
@@ -466,6 +466,64 @@ jobs:
             .tox/.coverage
             .tox/coverage.xml
 
+  combine-coverage:
+    if: ${{ success() || failure() }}
+    name: Combine coverage
+    needs: test
+    runs-on: ubuntu-latest-64-cores
+    steps:
```

Comment:
@sfc-gh-nkrishna does that mean there was no code coverage for snowpark python after we merge back?

Reply:
Snowpark Python used some third-party tools that were rate limiting us (and weren't very accurate), so we agreed with Snowpark Python team to remove that in favor of having our code coverage diff. See this Slack conversation: https://snowflake.slack.com/archives/C02KH7E6SHX/p1715797748524849